modules/livewrappedBidAdapter.js

```diff
@@ -177,14 +184,33 @@ function hasPubcid(bid) {
 }
 
 function bidToAdRequest(bid) {
-  return {
+  var adRequest = {
     adUnitId: bid.params.adUnitId,
     callerAdUnitId: bid.params.adUnitName || bid.adUnitCode || bid.placementCode,
     bidId: bid.bidId,
     transactionId: bid.transactionId,
-    formats: bid.sizes.map(sizeToFormat),
+    formats: getSizes(bid).map(sizeToFormat),
     options: bid.params.options
   };
+
+  if (bid.mediaTypes && bid.mediaTypes.banner && bid.mediaTypes.native) {
+    adRequest.banner = true;
+  }
+
+  if (bid.mediaTypes && bid.mediaTypes.native) {
+    adRequest.native = bid.mediaTypes.native;
+  }
+
+  return adRequest;
+}
+
+function getSizes(bid) {
+  if (typeof utils.deepAccess(bid, 'mediaTypes.banner.sizes') !== 'undefined') {
```

Comment:
I see you already are using deepAccess! nice :)

Can probably be changed to leave off the `typeof {} !== 'undefined'` maybe?

Since if it was not defined it will just return `undefined` which is falsely.

No big deal though.